hist/histpainter/src/TGraphPainter.cxx

```diff
@@ -633,18 +633,18 @@ graphically. Point will be highlighted as "point circle" (presented by
 marker object). Moreover, any highlight (change of point) emits signal
 `TCanvas::Highlighted()` which allows the user to react and call their own
 function. For a better understanding please see also the tutorials
-`$ROOTSYS/tutorials/visualisation/graphs/hlGraph*.C` files.
+`$ROOTSYS/tutorials/visualisation/graphs/gr*_highlight*.C` files.
 
 Highlight mode is switched on/off by `TGraph::SetHighlight()` function
 or interactively from `TGraph` context menu. `TGraph::IsHighlight()` to verify
 whether the highlight mode enabled or disabled, default it is disabled.
 
 ~~~ {.cpp}
-    root [0] .x $ROOTSYS/tutorials/visualisation/graphs/gerrors2.C
+    root [0] .x $ROOTSYS/tutorials/visualisation/graphs/gr003_err_2gr.C
     root [1] // try SetHighlight() interactively from TGraph context menu
 ~~~
 
-\image html hlgerrors2.gif "Highlight mode for graph"
+\image html hl_gr003_err_2gr.gif "Highlight mode for graph"
```

Comment:
`hl_gr003_err_2gr` seems a bit of a different naming convention . The macros were renamed  with "highlight" inside the name. Ex: gr301_highlight1.C